changestopiglatin.txt

```diff
@@ -0,0 +1,13 @@
+
+pyg = 'ay'
+
+original = raw_input('Enter a word:')
+
+if len(original) > 0 and original.isalpha():
+    word = original.lower()
+    first = word[0]
+    new_word = word+first+pyg
+    new_word = new_word[1:len(new_word)]
+    print new_word
```

Comment:
I would have written this entire big block as:
```
if len(original) > 0 and original.isalpha():
    print s[1:] + s[0] + 'ay'
else:
    ...
```
Python is nice because it lets you write complicated things really concisely.